pandas/tests/tseries/offsets/test_ticks.py

```diff
@@ -90,11 +90,9 @@ def test_tick_equality(cls, n, m):
     left = cls(n)
     right = cls(m)
     assert left != right
-    assert not (left == right)
 
     right = cls(n)
     assert left == right
-    assert not (left != right)
```

Comment:
Same comment as above, even though should be equivalent good to test both since different operators